# 探索使用大型语言模型分析长链非编码 RNA 转录调控的潜力和挑战

发布时间：2024年11月05日

`LLM应用` `基因调控`

> Exploring the Potentials and Challenges of Using Large Language Models for the Analysis of Transcriptional Regulation of Long Non-coding RNAs

# 摘要

> 关于长链非编码 RNA（lncRNAs）的研究因其在基因调控和疾病机制中的关键作用而受到了极大的关注。然而，lncRNA 序列的复杂性和多样性，以及对其功能机制和表达调控的有限了解，给 lncRNA 研究带来了重大挑战。鉴于大型语言模型（LLMs）在捕获序列数据中的复杂依赖关系方面取得了巨大成功，本研究旨在系统地探索 LLMs 在与 lncRNA 基因转录调控相关的序列分析中的潜力和局限性。我们广泛的实验表明，经过微调的基因组基础模型在逐渐复杂的任务中表现出了有希望的性能。此外，我们对任务复杂性、模型选择、数据质量和生物可解释性对 lncRNA 基因表达调控研究的关键影响进行了有见地的分析。

> Research on long non-coding RNAs (lncRNAs) has garnered significant attention due to their critical roles in gene regulation and disease mechanisms. However, the complexity and diversity of lncRNA sequences, along with the limited knowledge of their functional mechanisms and the regulation of their expressions, pose significant challenges to lncRNA studies. Given the tremendous success of large language models (LLMs) in capturing complex dependencies in sequential data, this study aims to systematically explore the potential and limitations of LLMs in the sequence analysis related to the transcriptional regulation of lncRNA genes. Our extensive experiments demonstrated promising performance of fine-tuned genome foundation models on progressively complex tasks. Furthermore, we conducted an insightful analysis of the critical impact of task complexity, model selection, data quality, and biological interpretability for the studies of the regulation of lncRNA gene expression.

[Arxiv](https://arxiv.org/abs/2411.03522)